extensions/wikia/AdEngine/js/TaboolaHelper.js

```diff
@@ -0,0 +1,49 @@
+/*global define*/
+define('ext.wikia.adEngine.taboolaHelper', [
+	'ext.wikia.adEngine.adContext',
+	'wikia.document',
+	'wikia.window'
+], function (adContext, document, win) {
+	'use strict';
+
+	var context = adContext.getContext(),
+		logGroup = 'ext.wikia.adEngine.provider.taboola',
```

Comment:
I suggest keeping the `logGroup` the same as the module name itself, in this case:
```
logGroup = 'ext.wikia.adEngine.taboolaHelper'
```
feel free to ping me or @gabrys to show you how the logging works! :smile_cat: